### PR TITLE
Only apply 3 monitor fix to eruption

### PR DESCRIPTION
**Pull request by MrMinos**
Edit: We have another three monitor setup at the lab that does not have middle monitor in a vertical setup. Would be nice to change the script to only apply on eruption which currently has vertical middle monitor setup.

### CODE REVIEW

**Comment by dkess** [comment]
Why is this necessary?

**Comment by abizer** [comment]
```python
import socket
hostname = socket.gethostname()
```

**Comment by MrMinos** [comment]
Alright tested on hurricane, which currently is on my environment, that this works.

File: modules/ocf_desktop/files/xsession/fix-displays [hunk 1]
```diff
@@ -22,8 +24,8 @@ if __name__ == '__main__':
         prev = monitor
 
     # special case: 3 monitors
```
Comment (kpengboy) [inline modules/ocf_desktop/files/xsession/fix-displays]: Don't forget to update comment

Comment (MrMinos) [reply modules/ocf_desktop/files/xsession/fix-displays]: Updated, thanks kpengboy!

**Comment by MrMinos** [comment]
Added in the commit `ed44001` to ensure the monitor ordering at OS level to follow the physical port ordering at the graphics card.

This commit may break the current double monitor setup if they were not setup by physical port ordering at the graphics card. We will have to ensure the port ordering before rolling out to master branch.

```
    # For example, with the below grahpics card, excluing DVI port
    # Left most monitor should go to the left most port (DP 1) and so on
    #   
    # +------------------------------------------------+
    # |   +----------------+    | +------------------+ |
    # |   |    D V I       |    | |  |  Exhaust   |  | |
    # |   +----------------+    | +------------------+ |
    # +------------------------------------------------+
    # |   +-------+ +--------+  +-------+   +-------+  |
    # |   +  DP 1 | |  HDMI  |  +  DP 2 |   +  DP 3 |  |
    # |    \------+ \-------/    \------+    \------+  |
    # +------------------------------------------------+
```

File: hieradata/type/desktop.yaml [hunk 2]
```diff
@@ -3,6 +3,7 @@ classes:
 
 ocf::auth::glogin: [[ocf, LOCAL]]
 ocf::auth::gsudo: [ocfstaff]
+ocf::auth::usudo: [minos]
```
Comment (cg505) [inline hieradata/type/desktop.yaml]: plz don't forget to remove this

**Comment by MrMinos** [comment]
Ah you caught me, I was trying to sneak into having root xD

**Comment by minos-cs184** [comment]
Today, after hours; we can merge this into master and if some of the hosts are not correctly oriented, I can go over those with my family members and show how fix-display works.

**Pull request merged**